packages/@aws-cdk/aws-appsync/test/verify.graphql.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
might be worth putting it into a tools/scripts folder if we're expecting more of them.
It will be neat to fold all this verification logic into a custom resource or something similar (not for this PR though!)

Reply:
ok ill do it in a follow up PR